fgpyo/sam/__init__.py

```diff
@@ -360,6 +361,46 @@ class _CigarOpUtil:
     }
 
 
+@dataclass(frozen=True)
+class RangeOfBases:
```

Comment:
This is clever but I still think returning a `tuple[int, int]` is simpler and easier for the user

Reply:
done.